recovery/mainwindow.cpp

```diff
@@ -215,6 +215,9 @@ MainWindow::MainWindow(const QString &drive, const QString &defaultDisplay, QSpl
     {
         _fixate = true;
     }
+
+    copywpa();
```

Comment:
Should this have a capital `W` ?

Reply:
Yep. Don't know where it went - it's there in my original code. Odd.